src/libs/KeyboardShortcut/index.js

```diff
@@ -66,15 +66,27 @@ document.addEventListener('keydown', bindHandlerToKeyupEvent);
  * The "subClass" is used by pages to bind /unbind with no worries
  */
 const KeyboardShortcut = {
+    /**
+     * Returns keyCode for a given key
+     * @param {String} key The key to watch, i.e. 'K' or 'Escape'
+     * @returns {Number} The key's keyCode, i.e. 75 or 27
+     */
+    getKeyCode(key) {
+        if (key === 'Escape') {
```

Comment:
How come `key.charCodeAt(0)` doesn't work for escape? Might be worth adding that as a comment.

Reply:
Thanks for adding the comment! Makes sense now. I think eventually this will probably need to be refactored so that a `key` with a length of 1 will have an early return, and then maybe a switch statement or a key to keyCode mapping for anything else, but this is fine for now. NAB